File: code/ClassPrivateMethodAccessInstance.java
class AnimalJ {
  private String name;

  public AnimalJ(String name) {
    this.name = name;
    greet();
  }

  private void greet() {
    System.out.println("Hello! My name is " + this.name);
  }
}

class ClassPrivateMethodAccessInstance {
  public static void main(String[] args) {
    new AnimalJ("Kelya");
  }
}

File: code/ClassPrivateMethodAccessInstance.php
<?php

class AnimalJ
{
    private $name;

    public function __construct(string $name)
    {
        $this->name = $name;
        $this->greet();
    }

    private function greet()
    {
        echo "Hello! My name is " . $this->name;
    }
}


new AnimalJ("Kelya");

